apps/events/permissions/events.py

```diff
@@ -7,6 +7,8 @@ class EventPermissions(permissions.BasePermission):
 
     def has_object_permission(self, request, view, obj):
         if request.method in permissions.SAFE_METHODS:
+            if view.action == "get_private_url":
+                return is_owner(request, obj)
             return True
```

Comment:
since we are removing get url, we can remove this

Reply:
we can add something like this:
```

    def has_private_token(self, request: HttpRequest, obj):
        return obj.type == 'private' and request.params.get('token') != obj.token
```

Reply:
if request.method in permissions.SAFE_METHODS:
            if obj.type == 'private' and not has_access_to_private(request, obj):
                return False
            return True